tests/features/common/kieserver/kieserver-common.feature

```diff
@@ -41,6 +41,16 @@ Feature: Kie Server common features
       | KIE_SERVER_HOST    | my-custon-host.example.com |
       | KIE_SERVER_PORT    | 80                         |
     Then container log should contain -Dorg.kie.server.location=http://my-custon-host.example.com:80/services/rest/server
+
+  Scenario: Test the KIE_SERVER_HOST configuration with custom host with default port
+    When container is started with env
+      | variable           | value                      |
+      | KIE_SERVER_HOST    | my-custon-host.example.com |
+    Then container log should contain -Dorg.kie.server.location=http://my-custon-host.example.com:80/services/rest/server
+
+  Scenario: Test the KIE_SERVER_HOST with no value provided
+    When container is ready
+    Then container log should contain :8080/services/rest/server
```

Comment:
In this case no, because the port 80 is the default when KIE_SERVER_HOST is set. In this case, KIE_SERVER_HOST is empty and will try to query the route name on kubernetes api which will fail, then it is defaulted to $HOSTNAME and port 8080.

Reply:
Ah gotcha. Thanks!